addon/components/perfect-scroll/component.js

```diff
@@ -82,14 +121,18 @@ export default Ember.Component.extend({
     }
   }).readOnly(),
 
+  _scrollElement: Ember.computed('eId', function() {
+    return document.getElementById(get(this, 'eId'));
+  }),
+
   /**
    * Binds perfect-scrollbar events to function
    * and then calls related events if user gives the action
    */
   bindEvents() {
     let self = this;
     let mapping = {};
-    let el = $(document.getElementById(get(this, 'eId')));
+    let el = document.getElementById(get(this, 'eId'));
 
     psEvents.map(evt => {
       mapping[evt] = function() {
```

Comment:
May be we can also extend here when we receive all events so you can filter out for `ps-scroll-y` and `ps-scroll-x`?